src/EA.Weee.Core/Validation/MinimumElementAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace EA.Weee.Core.Validation
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
```

Comment:
Can we change the attribute so that it takes a string that is the error message to display then we can re-use easily elsewhere